atoma-proxy/src/server/handlers/chat_completions.rs

```diff
@@ -692,6 +700,7 @@ async fn handle_streaming_response(
         .await
         .map_err(|e| AtomaProxyError::InternalError {
             message: format!("Error sending request to inference service: {e:?}"),
+            client_message: Some("Failed to connect to the node.".to_string()),
```

Comment:
Should we send the reason back to the client ? Or can we add more contex t?

Reply:
You want to send the actual error? I mean the connection failed (dropped, timed out, etc.) I don't think they need to know that. It's just there is something wrong with the connection.